linux_os/guide/system/permissions/restrictions/sysctl_user_max_user_namespaces/rule.yml

```diff
@@ -0,0 +1,26 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora,ol8
+
+title: 'Disable the use of user namespaces'
+
+description: |-
+    {{{ describe_sysctl_option_value(sysctl="user.max_user_namespaces", value="0") }}}
+    When containers are deployed on the machine, the value should be set
+    to large non-zero value.
+
+rationale: |-
+    User namespaces are used primarily for Linux containers. The value 0
+    effectively disallows the use of user namespaces.
```

Comment:
```suggestion
   disallows the use of user namespaces.
```

Reply:
What is the nuance of saying "effectively disables" vs "disables"?

Reply:
Removed.